static/scss/navbar.scss

```diff
@@ -243,13 +239,13 @@
   }
 
   .Select-control {
-    border: 1px solid $program_selector_border !important;
+    border: none !important;
     box-shadow: none !important;
     cursor: pointer;
   }
 
   .Select-menu-outer,.Select-control,.Select-value-label,.Select-menu-outer,.Select-option {
-    background-color: $mm-brand-bg !important;
+    background-color: rgba(0,0,0,.1);
```

Comment:
I think we should generally avoid inline colors and try to declare them all as variables - could you declare this in the `_variables.scss` file? (same for the other inline colors)